Signal/src/ViewControllers/ConversationView/Cells/OWSContactOffersCell.m

```diff
@@ -38,16 +41,17 @@ - (void)commontInit
 {
     OWSAssert(!self.titleLabel);
 
-    //    [self setTranslatesAutoresizingMaskIntoConstraints:NO];
+    self.preservesSuperviewLayoutMargins = NO;
+    self.contentView.preservesSuperviewLayoutMargins = NO;
+    self.layoutMargins = UIEdgeInsetsZero;
+    self.contentView.layoutMargins = UIEdgeInsetsZero;
```

Comment:
Converted this cell to use auto layout.

Reply:
We want to layout with regard to the cell _edges_, not it's margins.  Our design uses non-standard margins.  This is the default value and redundant with our setting the layout margins to zero as we do here, so I'll remove them - the intent was to make our intention clear (use non-standard layout with respect to edges) and future-proof us against future changes.